src/arm/am572x-bone-common-univ.dtsi

```diff
@@ -0,0 +1,2505 @@
+/*
```

Comment:
I don't know that this is 'univ', since it doesn't include the cape-universal pinmux helpers. Does it have anything other than pinmux settings? What about -pinmux?

Reply:
So, we have this file `src/arm/am335x-bone-common-univ.dtsi` for BBB and I just created a similar one for BBAI here. It is basically the pin muxing part yeah and It makes the base bealgeboneai file cleaner. Also, It will be easier to merge changes into 5.4 kernel if we create this separate file.